apps/nuxt/components/blocks/layout/TheFooter.vue

```diff
@@ -0,0 +1,240 @@
+<template>
```

Comment:
Please make the bottom bar full width.
<img width="1725" alt="image" src="https://github.com/vuestorefront/nuxt3-magento-sdk-storefront/assets/45824492/961df80c-5f86-40d3-950a-b16d6fa78128">

Reply:
done

Reply:
Done 

<img width="1727" alt="image" src="https://github.com/vuestorefront/nuxt3-magento-sdk-storefront/assets/40073926/f85465db-8325-49f8-95c1-ad0ea30d31fe">

Reply:
Great, thanks 😄